source/documentation/doc-Introduction_to_the_VM_Portal/topics/Connecting_to_virtual_machines.adoc

```diff
@@ -0,0 +1,18 @@
+=== Connecting to virtual machines
+
+*Connecting to a virtual machine*
+
+. In the virtual machines pane, click *Run* in a virtual machine's card to start that virtual machine.
+
+. Click *Console* to connect to the virtual machine.
+
+. You will be asked to download a *.vv* file.
+
+. Open the file with *remote-viewer*. A console window displays.
++
+You can now use the virtual machine in the same way you would use a physical desktop.
+
+[NOTE]
+====
+To allow other users to connect to the VM, make sure you restart the virtual machine when you are finished using the console. Alternatively, the administrator can *Disable strict user checking* to eliminate the need for reboot between users. See link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#Virtual_Machine_Console_settings_explained[Virtual Machine Console Settings Explained] in the _Virtual Machine Management Guide_ for more information.
```

Comment:
'restart' may lead users to only reboot the VM which is not sufficient for connecting to the VM console. How about stating 'make sure you shut down and restart the virtual machine when you  are finished using the console'.

Reply:
Changes made.